airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -220,6 +226,9 @@ def __init__(
         if self.custom_image and self.image_version:
             raise ValueError("The custom_image and image_version can't be both set")
 
+        if self.custom_image_family and self.custom_image:
```

Comment:
just for sanity @ashishpatel0720 can you set `custom_image_family` and `image_version` together?

Reply:
yeah, make sense @marcosmarxm.
I will make the change.

Reply:
- [x]  sanity check added for custom_image_family and image_version